integration/fixtures_test.go

```diff
@@ -1,3 +1,11 @@
 package integration_test
 
-var testCatalogSourceImage = "quay.io/operatorhubio/catalog:latest"
+import "time"
+
+var (
+	referenceAddonCatalogSourceImageWorking = "quay.io/osd-addons/reference-addon-index@sha256:58cb1c4478a150dc44e6c179d709726516d84db46e4e130a5227d8b76456b5bd"
+	referenceAddonCatalogSourceImageBroken  = "quay.io/osd-addons/reference-addon-index@sha256:9e6306e310d585610d564412780d58ec54cb24a67d7cdabfc067ab733295010a"
```

Comment:
needs a doc block what failure scenario is prepared under this digest, so it's easy for others to reuse for other failure cases.

Reply:
added!